### ISSUE TITLE
Zombie Diagnostics

### ISSUE THREAD

**Issue by pappasam** [gianarb/coc-grammarly#18]
Diagnostics do not go away. For example, when a piece of text is changed, its related diagnostic message stays around longer than it should (presumably to eat brains).

I'm using the latest local development build with Neovim Nightly:

![bad-grammarly](https://user-images.githubusercontent.com/3723671/112067179-e01ccf00-8b24-11eb-99c5-bc70137e1fed.gif)

**Comment by gianarb** [comment]
Yeah I noticed that they take a bit too long but they disappear when you save, is it the same for you as well?

**Comment by TonalidadeHidrica** [comment]
I've just starting this awesome extension but it's a pity that I'm facing the same issue too.  The zombie diagnostics do not go away even after I saved the document, and I have to `:CocRestart` all the time to get rid of them.

**Comment by paulkre** [comment]
They don't go away for me either. Even after saving..
